Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -0,0 +1,438 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+
+using UnityEngine;
+using System.Collections;
+using System.Net;
+using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Function used to communicate with the device through the REST API
+    /// </summary>
+    public class BuildDeployPortal
+    {
+        // Consts
+        public const float kTimeOut = 6.0f;
+        public const int kTimeoutMS = (int)(kTimeOut * 1000.0f);
+        public const float kMaxWaitTime = 20.0f;
+
+        public static readonly string kAPI_ProcessQuery = @"http://{0}/api/resourcemanager/processes";
```

Comment:
Yes, it currently requires people to first...
1. pair the device. 
2. disabling HTTPS requirement in the device portal (or always connect local).